### PR TITLE
[16.0][MIG] account_move_fiscal_month: Migration to 16.0

### PR DESCRIPTION
**Pull request by BT-anieto**
Depends on #1708

### CODE REVIEW

**Review by BT-nfrancisco — APPROVED** [review]

**Review by BT-jcolmeiro — APPROVED** [review]
LGTM :+1:

**Comment by BT-nfrancisco** [comment]
@OCA/accounting-maintainers Hello. Why is this PR always building? Can we somehow push it? Thank you very much

**Comment by etobella** [comment]
Try to rebase, but I would recommend to fix tests first

**Comment by BT-anieto** [comment]
Thank you @etobella ! I don't get why the test are failing. They pass in my local. Do you have any idea? It seems more a timeout problem

**Comment by BT-anieto** [comment]
Thank you very much, @etobella

File: account_move_fiscal_month/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -1 +1,2 @@
 * Benjamin Willig <benjamin.willig@acsone.eu>
+* * Alberto Nieto de Pablos <alberto.nieto@braintec.com> (https://braintec.com)
```
Comment (etobella) [inline account_move_fiscal_month/readme/CONTRIBUTORS.rst]: Why did you add two asterisks?

**Comment by BT-anieto** [comment]
Hi @etobella Could we merge it?

**Review by etobella — APPROVED** [review]
/ocabot merge nobump
